low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -271,12 +271,22 @@ def process(self):
 
 class TestCmd(Cmd):
     """Starts test based on plan:
-    (sanity|alerts|self_primary|self_secondary).
+    (sanity|alerts|dev_sanity|full|performance|regression|sanity).
```

Comment:
`sanity` is repeated my mistake I guess.

Reply:
Yeah, corrected it.